dist/angular-validation.js

```diff
@@ -379,7 +379,7 @@
       var validCallback = $parse('success');
       var messageElem;
 
-      if (attrs.messageId) messageElem = angular.element(document.querySelector('#' + attrs.messageId));
+      if (attrs.messageId || attrs.validationGroup) messageElem = angular.element(document.querySelector('#' + (attrs.messageId || attrs.validationGroup)));
```

Comment:
What about define 
```
var validationGroup = attrs.validationGroup
```

so that we use validationGroup instead of repeatly calling attrs.validationGroup.